src/platform/utilities/sso/keepAliveSSO.js

```diff
@@ -34,9 +34,30 @@ export default async function keepAlive() {
       }[resp.headers.get('va_eauth_csid')],
     };
   } catch (err) {
+    const caughtExceptions = {
```

Comment:
@AGarcia-Clarity can we make the mapping a const on the module, so its just initialized once and not every time the catch block executes?